src/Input/InputEvents.cpp

```diff
@@ -198,6 +198,18 @@ InputEvents::drawButtons(Mode mode, bool full)
     : NULL;
 
   ButtonLabel::Set(menu, overlay_menu, full);
+
+  // Signal GlueMapWindow that buttons have been drawn
+  GlueMapWindow *map = CommonInterface::main_window->GetMapIfActive();
+  if (map != nullptr){
+      if (mode != MODE_DEFAULT){
+          map->SetBottomMargin(vertical_button_scale);
+          fprintf(stderr, "Setting margin!\n");
+      } else {
+          map->SetBottomMargin(0);
+          fprintf(stderr, "RESetting margin!\n");
+      }
+  }
```

Comment:
What does this code do, and wat does it have to do with "moving scale magic number to const"?

Reply:
When the `MenuBar` is drawn, this will signal call `SetBottomMargin(...)`  with either a margin when a mode other than `MODE_DEFAULT` is active or a with zero to reset the margin when default mode is (re)entered.

This has indeed nothing to do with "moving scale magic number to const". It seems I made a mistake when cleaning up my branch. This should have been in the "Set bottom margin when buttons are drawn" commit instead of the removing of printfs that happened there.